website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -1,500 +1,102 @@
 ---
-title: Golang Based Operator Quickstart
-linkTitle: Quickstart
-weight: 10
+title: Quickstart for Go-based Operators
+linkTitle: QuickStart
+weight: 20
+description: A simple set of instructions that demonstrates the basics of setting up and running a Go-based operator.
 ---
 
-**NOTE:** For the SDK versions prior to `v0.19.0` please consult the [legacy docs][legacy_quickstart_doc] for the [legacy CLI][legacy_CLI] and project.
-
-This guide walks through an example of building a simple memcached-operator using the operator-sdk CLI tool and controller-runtime library API.
+This guide walks through an example of building a simple memcached-operator powered by Go using tools and libraries provided by the Operator SDK
```

Comment:
Nit: 

Suggestion for more clarity: 
"...building a simple memcached-operator powered by Go using tools and libraries provided by the Operator SDK"

"...building a simple memcached-operator powered by Go, and using tools and libraries provided by the Operator SDK"

Reply:
@joelanford any suggestion here? I understand that the above suggesting would be replace it for `This guide builds a simple memcached-operator powered by Go using tools and libraries provided by the Operator SDK`. Are you ok with?

Reply:
Description is: 
```
A simple set of instructions that demonstrates the basics of setting up and running a Go-based operator.
```

IMO, this should remain as:
```
This guide walks through an example of building a simple memcached-operator using the operator-sdk CLI tool and controller-runtime library API.
```

I think these complement each other well. Alternatively, we could just remove this line completely.

Reply:
I'm trying to come up with a line that adds something helpful but doesn't duplicate the description, and I can't. I think just remove this line.

Reply:
I pushed my suggestion. I also think that how less verbosely we are be able to be better it would be.